### ISSUE TITLE
Problem with the finetuned model

### ISSUE THREAD

**Issue by colincqian** [Arthur151/ROMP#109]
Dear Arthur151, 

I tried to finetuned the ROMP_HRNet32 model with your provided configuration. The training process seems to converge perfectly and evaluation result is good. However, I notice a bias in all the rendered the result with romp.predict.image. I checked romp.predict.image with your provided model (ROMP_HRNet32_V1.pkl), and it does not have this issue. I am little confused with this result. Though I have added some new loss term, I have already set the loss weight to be zero to rule out the influence. 

Below, are one of  rendered result and tensorboard info:
![tensorboard](https://user-images.githubusercontent.com/44220675/144751597-28b30e23-0602-4ed8-98bb-2f7f2321faf9.png)
![image_00000](https://user-images.githubusercontent.com/44220675/144751608-8cc6aa3f-498b-430e-a9bd-36f0e5a7342f.jpg)

Thank you in advance for any suggestion to solve this issue.

**Comment by Arthur151** [comment]
Thanks for trying ROMP.
To fix this mis-alingment, please switch this configuration:
`smpl_mesh_root_align: False`
in config ymls.

**Comment by Arthur151** [comment]
@colincqian Feel free to tell me if there are anything wrong. I will try to solve it as soon as possible.

**Comment by colincqian** [comment]
Thanks for your advice. That seems to solve the rendered issue. I think it is just an issue from rendering the result, for the images render in result_image_train seems fine.

**Comment by Arthur151** [comment]
Thanks for the bug report. At my service! 

Best,
Yu
